test/cql-pytest/cassandra_tests/validation/operations/alter_test.py

```diff
@@ -88,6 +88,26 @@ def testDropWithTimestamp(cql, test_keyspace):
                [1, 4, 4, 4],
                [1, 100, 100, 100])
 
+def testDropWithTimestampPrepared(cql, test_keyspace):
```

Comment:
Please don't add tests to cql-pytest/cassandra_tests - this is where only translated Cassandra tests live. You can add a new file a directory above, like cql-pytest/test_alter_table.py. Please also note that the tests outside cassandra_tests use fewer utility functions (e.g., use `cql.execute()` instead of the utility function `execute(cql, ...)` which I added to make Cassandra tests easier to translate).
(see  test/cql-pytest/README.md with more about the philosophy of these tests, including avoiding the cassandra_tests directory.

Reply:
By the way, the test looks good. Please make sure you also run it with Cassandra (if you don't know how, it's easy and I recently added clearer instructions in test/cql-pytest/README.md) to be sure we handle the prepared statement version like Cassandra does.

Reply:
I've run the test on cassandra doesn't allow `?` as a timestamp in ALTER TABLE ... DROP ... USING TIMESTAMP ...
Added a `scylla_only` to the test.